projects/rocprofiler-systems/source/bin/rocprof-sys-sample/impl.cpp

```diff
@@ -373,11 +373,19 @@ parse_args(int argc, char** argv, std::vector<char*>& _env)
             if(_v.size() > 1) update_env(_env, "ROCPROFSYS_OUTPUT_PREFIX", _v.at(1));
         });
     parser
-        .add_argument({ "-T", "--trace" }, "Generate a detailed trace (perfetto output)")
+        .add_argument({ "-T", "--trace" },
+                      "Generate a detailed trace (perfetto output) from cached data")
         .max_count(1)
         .action([&](parser_t& p) {
             update_env(_env, "ROCPROFSYS_TRACE", p.get<bool>("trace"));
         });
+    parser
+        .add_argument({ "-L", "--trace-legacy" },
+                      "Generate a detailed trace (perfetto output)")
+        .max_count(1)
```

Comment:
We probably also want to add a conflict, so we can't use "trace" and "trace-legacy" at the same time.
```suggestion
        .max_count(1)
        .conflicts({"trace"})
```

Reply:
I could, at the moment I don't see the use case where both options are needed. But in general, it's designed in such way that they won't collide with each other.